dev-tools/ecs-migration.yml

```diff
@@ -54,6 +54,11 @@
   alias6: true
   alias: true
 
+- from: read_timestamp
```

Comment:
You put it here "globally" but in the fields.yml you added it to each Beat. Is that on purpose? Otherwise I would like to keep here in this file both mention as I plan to also add the Beat name to each entry. Like this we can potentially generate some docs per Beat that show which fields changed.

Reply:
It's true that the only two field definitions I saw were Filebeat and Journalbeat. However if we want a script to do the right thing based on ecs-migration.yml, we would have to have the YAML (not the comments) specify this scope.

What do you think of this?

```YAML
- from: read_timestamp
  to: event.created
  alias6: true
  alias: true
  beats:
    - filebeat
    - journalbeat
```

Or `beats:` => `scope:`?

Reply:
yes, had something similar in mind. My idea was to duplicate entries and just have `beat: filebeat` and in the other block `beat: journalbeat`. Like this we can keep the blocks in the yaml file together for each Beat and not some outside.

Reply:
Makes total sense. Let's go with that!